rosdep/base.yaml

```diff
@@ -2541,6 +2551,136 @@ libicu-dev:
   fedora: [libicu-devel]
   gentoo: [dev-libs/icu]
   ubuntu: [libicu-dev]
+libignition-cmake2-dev:
+  debian:
+    buster: [libignition-cmake2-dev]
+  ubuntu:
+    focal: [libignition-cmake2-dev]
+libignition-common3:
+  debian:
+    buster: [libignition-common3]
+  ubuntu:
+    focal: [libignition-common3]
+libignition-common3-dev:
+  debian:
+    buster: [libignition-common3-dev]
+  ubuntu:
+    focal: [libignition-common3-dev]
+libignition-fuel-tools4:
+  debian:
+    buster: [libignition-fuel-tools4]
+  ubuntu:
+    focal: [libignition-fuel-tools4]
+libignition-fuel-tools4-dev:
+  debian:
+    buster: [libignition-fuel-tools4-dev]
+  ubuntu:
+    focal: [libignition-fuel-tools4-dev]
+libignition-gazebo3:
+  debian:
+    buster: [libignition-gazebo3]
+  ubuntu:
+    focal: [libignition-gazebo3]
+libignition-gazebo3-dev:
+  debian:
+    buster: [libignition-gazebo3-dev]
+  ubuntu:
+    focal: [libignition-gazebo3-dev]
+libignition-gui3:
+  debian:
+    buster: [libignition-gui3]
+  ubuntu:
+    focal: [libignition-gui3]
+libignition-gui3-dev:
+  debian:
+    buster: [libignition-gui3-dev]
+  ubuntu:
+    focal: [libignition-gui3-dev]
+libignition-launch2:
+  debian:
+    buster: [libignition-launch2]
+  ubuntu:
+    focal: [libignition-launch2]
+libignition-launch2-dev:
+  debian:
+    buster: [libignition-launch2-dev]
+  ubuntu:
+    focal: [libignition-launch2-dev]
+libignition-math6:
+  debian:
+    buster: [libignition-math6]
+  ubuntu:
+    focal: [libignition-math6]
+libignition-math6-dev:
+  debian:
+    buster: [libignition-math6-dev]
+  ubuntu:
+    focal: [libignition-math6-dev]
+libignition-math6-eigen3:
+  debian:
+    buster: [libignition-math6-eigen3]
+  ubuntu:
+    focal: [libignition-math6-eigen3]
```

Comment:
This one doesn't work for me on Focal. @nuclearsandwich @j-rivero does this package need to be imported into the ros apt repo too?

```console
$ apt install -s $(rosdep resolve libignition-math6-eigen3 | grep -v -e "^#")
NOTE: This is only a simulation!
      apt needs root privileges for real execution.
      Keep also in mind that locking is deactivated,
      so don't depend on the relevance to the real current situation!
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libignition-math6-eigen3
```

With these apt repos

```console
$ apt-get update
Get:1 http://security.ubuntu.com/ubuntu focal-security InRelease [107 kB]
Get:2 http://packages.ros.org/ros/ubuntu focal InRelease [4,650 B]            
Get:3 http://packages.ros.org/ros2-testing/ubuntu focal InRelease [3,931 B]               
Hit:4 http://archive.ubuntu.com/ubuntu focal InRelease                                     
Get:5 http://packages.ros.org/ros/ubuntu focal/main amd64 Packages [199 kB]    
Get:6 http://archive.ubuntu.com/ubuntu focal-updates InRelease [107 kB]                 
Get:7 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [13.2 kB]
Get:8 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [134 kB]                            
Get:9 http://packages.ros.org/ros2-testing/ubuntu focal/main amd64 Packages [307 kB]    
Get:10 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [42.3 kB]         
Get:11 http://archive.ubuntu.com/ubuntu focal-backports InRelease [98.3 kB]                                               
Get:12 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [137 kB]
Get:13 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [13.2 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [252 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [2,900 B]
Fetched 1,420 kB in 2s (899 kB/s)                          
Reading package lists... Done
```

Reply:
Sorry, my bad, even in the OSRF repo, these only come in `-dev` form, fixed in 
ee4fcd5 

References:

* http://packages.osrfoundation.org/gazebo/ubuntu-stable/pool/main/i/ignition-math6/
* http://packages.ros.org/ros/ubuntu/pool/main/i/ignition-math6/